test/controllers/UploadControllerSpec.scala

```diff
@@ -109,6 +109,83 @@ class UploadControllerSpec extends FrontEndTestHelper {
       contentAsString(uploadPage) must include("Uploading records")
       contentAsString(uploadPage) must include("Your upload is complete and has been saved")
     }
+
+    "redirect to the transfer agreement page if the transfer agreement for that judgment has not been signed" in {
```

Comment:
Should this test be: "... for that judgment has not been **agreed**" to make it consistent with the other tests.